launch/seher_demo.launch

```diff
@@ -1,9 +1,10 @@
 <?xml version="1.0"?>
 <launch>
 
- <arg name="prompts" default="true" />
+ <arg name="prompts" default="false" />
  <arg name="robot" default="ur5e" />
  <rosparam file="$(find ur_manipulation)/config/$(arg robot).yaml"/>
- <node name="seher_demo" pkg="ur_manipulation" type="seher_demo" respawn="false" output="screen" args="$(arg prompts)"/>
+ <node name="endurance_demo" pkg="ur_manipulation" type="endurance_demo"
```

Comment:
This references the endurance_demo node in this package, which does not exist?? In general, I would say keep this PR only for the "fixed format" commit and drop the commit "applicable to prbt and logging with epoch time" from here, which you can move to your other PR.  Also, please rewrite the description of the PR to reflect the same.